steps/microsoft-teams-integration/1.2.1/step.yml

```diff
@@ -0,0 +1,145 @@
+title: Microsoft Teams Integration
+summary: Allows you to recieve build status updates on Microsoft Team using Incoming
+  Webhooks
+description: |
+  Uses Incoming Webhooks feature in Microsft Teams to send updates on your builds
+website: https://github.com/amrfarid140/bitrise-step-microsoft-teams-integration
+source_code_url: https://github.com/amrfarid140/bitrise-step-microsoft-teams-integration
+support_url: https://github.com/amrfarid140/bitrise-step-microsoft-teams-integration/issues
+published_at: 2021-02-06T21:58:08.132756+11:00
+source:
+  git: https://github.com/amrfarid140/bitrise-step-microsoft-teams-integration.git
+  commit: 9da9a2007cbaa7c79928ce52ba749248cb8c5bf3
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- notification
+toolkit:
+  go:
+    package_name: github.com/amrfarid140/bitrise-step-microsoft-teams-integration
+deps:
+  brew:
+  - name: go
```

Comment:
Go should not be a dependency if go toolkit is used: https://discuss.bitrise.io/t/golang-as-a-dependency/6989

Reply:
can I add the change to this PR or do I need to submit as a new PR?

Reply:
According to the current guidelines, there should be only one commit for PR (except icon).
It is up to you if you prefer a force push or a new PR, usually a new PR is raised.

Reply:
no worries will close this and open a new PR with the changes, thanks @koral--